src/mtstate.c

```diff
@@ -23,14 +23,20 @@
 #include "mtstate.h"
 #include "trig.h"
 
+#ifdef DEBUG_MTSTATE
+# define LOG_DEBUG_MTSTATE LOG_DEBUG
+#else
+# define LOG_DEBUG_MTSTATE LOG_DISABLED
+#endif
+
 static int inline percentage(int dividend, int divisor)
 {
-	return (double)dividend / (double)divisor * 100;
+	return (int)((double)dividend / (double)divisor * 100);
```

Comment:
Wouldn't it be implicitly casted?

Reply:
- To silence errors when compiling with -Wfloat-conversion
- Generally better practice to cast explicitly

Reply:
Thanks for the review! I'll make changes soon.